src/main/java/io/cryostat/recordings/RecordingArchiveHelper.java

```diff
@@ -890,6 +913,27 @@ public void validateSourceTarget(String sourceTarget)
         }
     }
 
+    // Returns serviceUri associated with jvmId
+    public String validateJvmId(String jvmId) throws JvmIdDoesNotExistException {
```

Comment:
Method name seems a little misleading, this isn't "validating" the ID so much as doing a reverse lookup.

Reply:
On that note I also have a reverse lookup implementation that looks like it provides a similar contract here:

https://github.com/cryostatio/cryostat/blob/2c5ac7f91c0a891c32cbd96509f823c3668e1563/src/main/java/io/cryostat/recordings/JvmIdHelper.java

from https://github.com/cryostatio/cryostat/pull/1188